pulp_smash/tests/rpm/api_v2/test_unavailable_checksum.py

```diff
@@ -84,13 +84,22 @@ class UpdatedChecksumTestCase(unittest.TestCase):
     Do the following:
 
     1. Create, sync and publish a repository with checksum type "sha256".
-    2. Assert that the checksum type is the only one present on the
-       ``primary.xml``.
+    2. After that several XML files - part of repodata will be inspected.
+       ``primary.xml``  - Assert that the checksum type is the only
+       one present.
```

Comment:
This text is line-wrapped. Semantically, there is no break of any kind between the end of this sentence and the beginning of the next. reStructuredText offers semantic constructs for doing this, like definition lists:

```
term one
    Definition of term one.
term two
    Definition of term two.
```

They could be adapted to this situation.

Reply:
Updated per comments.

Reply:
Try one more time. Here's the documentation for reST [definition lists](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists).